src/stores/AppStore.ts

```diff
@@ -1304,19 +1304,18 @@ export class AppStore {
             this.apiService.setToken(authTokenParam);
         }
 
-        autorun(() => {
+        autorun(async () => {
             if (this.astReady && this.zfpReady && this.cartaComputeReady && this.apiService.authenticated) {
+                await this.connectToServer();
```

Comment:
Do Async functions work ok with autorun?

Reply:
thanks for spotting this, fixed & please have a look.(ref: https://github.com/mobxjs/mobx/issues/1003)